### DISCUSSION TITLE
Changing the line style for specific segments in a line chart

### DISCUSSION THREAD

**Discussion by JulesLassara** [GSI-CS-CO/chart-fx#512] [General]
I am looking for changing the line steel for a specific segment of the data set. Do you have any idea to do that ?
Is there another way than using FragmentedDataSet ? To change the markers color, I applied a `addDataStyle(index, style)` on the data set, but is there anything similar for a part of the line ? I can't find it.

**Comment by RalphSteinhagen** [comment]
@JulesLassara thanks for your interest in our little lib. For the time being, there are only the options for customising individual data points and not line segments. The latter is a bit tricky and your idea of using the FragmentedDataSet seems to be a good choice. 

I am wondering what a performant/robust API for such a use-case could look like for non-sorted data points/segments. :thinking: 

Any ideas, suggestions, and PRs with -- for example -- a new renderer (or extension) that can do this would be welcome.
